code/style.css

```diff
@@ -2,6 +2,226 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
```

Comment:
This file could need some more love! There's some empty rows that makes it harder to read (but sometime it helps!) and the indention is not always the same everywhere